RELEASE_NOTES.md

```diff
@@ -1,5 +1,90 @@
 This file holds "in progress" release notes for the current release under development and is intended for consumption by the Chef Documentation team. Please see <https://docs.chef.io/release_notes.html> for the official Chef release notes.
 
+# Chef Infra Client 15.4
+
+## converge_if_changed Improvements
+
+Chef Infra Client will now take into account any `default` values specified in custom resources when making converge determinations with the `converge_if_changed` helper. Previously default values would be ignored which caused necessary changes to be skipped. Note: This may cause behavior changes for some users, but we believe this change is an impacting bug for enough users to make it outside of a major release. Thanks [@ jakauppila](https://github.com/jakauppila) for reporting this.
+
+## Bootstrap Improvements
+
+Several improvements have been made to the `knife bootstrap` command to make it more reliable and secure:
+
+- File creation is now wrapped in a umask to avoid potential race conditions
```

Comment:
Torn between my suggestion below and what everyone else uses when referring to a singular umask.
```suggestion
- File creation is now wrapped in an umask to avoid potential race conditions
```